src/test/incremental/hashes/let_expressions.rs

```diff
@@ -38,6 +38,20 @@ pub fn change_name() {
 #[rustc_clean(label="HirBody", cfg="cfail3")]
 #[rustc_metadata_clean(cfg="cfail2")]
 #[rustc_metadata_clean(cfg="cfail3")]
+#[rustc_dirty(label="MirValidated", cfg="cfail2")]
+#[rustc_clean(label="MirValidated", cfg="cfail3")]
+#[rustc_dirty(label="MirOptimized", cfg="cfail2")]
+#[rustc_clean(label="MirOptimized", cfg="cfail3")]
+#[rustc_clean(label="TypeckTables", cfg="cfail2")]
```

Comment:
@michaelwoerister isn't it a bit surprising that `TypeckTables` is clean here? (Am I correct to interpret clean as green?)

Reply:
Oh, I guess it's just that the name of the variable doesn't show up in there.

Reply:
Yes, it is surprising but it's valid, I think.